src/compile/legend/parse.ts

```diff
@@ -167,8 +167,8 @@ export function parseLegendForChannel(model: UnitModel, channel: NonPositionScal
         ...(selections?.length && isFieldDef(fieldOrDatumDef)
           ? {name: `${varName(fieldOrDatumDef.field)}_legend_${part}`}
           : {}),
-        ...(selections?.length ? {interactive: !!selections} : {}),
-        update: value,
+        ...(selections?.length ? {interactive: true} : {}),
```

Comment:
Checking my understanding - Should this be `interactive: true` even if there are no `selections` ?